scapy/layers/dot11.py

```diff
@@ -1664,7 +1876,8 @@ class Dot11CCMP(Dot11Encrypted):
 bind_layers(Dot11, Dot11Disas, subtype=10, type=0)
 bind_layers(Dot11, Dot11Auth, subtype=11, type=0)
 bind_layers(Dot11, Dot11Deauth, subtype=12, type=0)
-bind_layers(Dot11, Dot11Ack, subtype=13, type=1)
+bind_layers(Dot11, Dot11Action, subtype=13, type=0)
+bind_layers(Dot11, Dot11Ack, subtype=12, type=1)
```

Comment:
This is incorrect, subtype should be 13.
Could you, at this occasion, add a unit test for Dot11Ack ? :P
Also could you (same, opportunistically), add
```
# 802.11-2016 9.2.4.1.3 Type and Subtype subfields
```
before the list of `bind_layers(Dot11, ..., subtype=..., type=...)` ?

Thanks !

Reply:
Thanks for the review. That is my mistake and its fair! I can add a unit test for `Dot11Ack`. However,

Ack frame does not contain a payload, hence I am not sure how to test it.

For example see this ack frame:
```
b'\x00\x008\x00/@@\xa0 \x08\x00\xa0 \x08\x00\x00\xc8\x85\x99B\x00\x00\x00\x00\x10\x18<\x14@\x01\xd5\x00\x00\x00\x00\x00\x00\x00\x00\x00\xc6\xe5\x99B\x00\x00\x00\x00\x16\x00\x11\x03\xd3\x00\xd5\x01\xd4\x00\x00\x00`\xf4E\x99];\xcf_&\xe0'
```
![image](https://user-images.githubusercontent.com/7614296/210257273-e406412a-7ef2-474a-8164-db1244a1e1b3.png)
![image](https://user-images.githubusercontent.com/7614296/210257456-9af1a5ad-d9ec-4b8d-9351-5115a9e0c7cf.png)

`Dot11Ack` layer is not used at all 🤔. For the sake of completeness, having `Dot11Ack` layer may make sense. But beyond that I am not sure about its usefulness.

Scapy is still new to me and I don't know much about its inner workings, so please correct me if I am missing something.

Reply:
Something like this is enough:
```python
pkt = Dot11(bytes(Dot11()/Dot11Ack()))
assert pkt.subtype == 13
assert pkt.type == 1
```